<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinaryOptionsWidget</class>
 <widget class="QWidget" name="BinaryOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>240</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="lSkipBytes">
     <property name="text">
      <string>Skip Bytes:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="niVectors">
     <property name="toolTip">
      <string>The number of vectors in the file</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lDataType">
     <property name="text">
      <string>Data type:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbDataType"/>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="sbSkipStartBytes">
     <property name="toolTip">
      <string>Skip bytes at start of file</string>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="sbSkipBytes">
     <property name="toolTip">
      <string>Skip bytes after each value</string>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lVectors">
     <property name="text">
      <string>Vectors:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lSkipStartBytes">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Skip start bytes:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lByteOrder">
     <property name="text">
      <string>Byte order:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbByteOrder"/>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>74</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chbCreateIndex">
     <property name="text">
      <string>Create index column</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
